README.md

```diff
@@ -18,7 +18,6 @@ This app aims to help KiwiHarvest’s volunteers keep track of all off the super
 + Helps ops confirm when a delivery is complete by drivers en route
 + Tracks the weight of all the pickups and dropoffs for the day
 
-
```

Comment:
Two blank lines between header sections actually isn't a bad idea. EDA does it in _almost_ all of our Markdown.